data/projects/extend_linter_integration.md

```diff
@@ -1,21 +1,25 @@
 A very easy way to cover a broad spectrum of analysis routines
-is wrapping linters into coala. Currently we have the `@linter` decorator
-which simplifies this integration. Based on this decorator, a large selection
-of linters can be implemented rapidly. 
-
-This project would widen the range of existing linters coala supports and makes 
-them available for usage in coala. This helps devs that had their favourite tools
-not supported before. It also enhances the testing API of linter bears, which are
-essential to a bugfree codebase. A tutorial on writing linter bears is
-available at [the linter documentation](http://api.coala.io/en/latest/Developers/Writing_Linter_Bears.html).
+is wrapping linters into coala.
+Currently we have the `@linter` decorator which simplifies this integration.
+Based on this decorator, a large selection of linters can be implemented
+rapidly.
+
+This project would widen the range of existing linters coala supports and makes
+them available for usage in coala.
+This helps devs whose favourite tools were not supported.
+It also enhances the testing API of linter bears, which are essential to a
+bugfree codebase.
+A tutorial on writing linter bears is available at
+[the linter documentation](http://api.coala.io/en/latest/Developers/Writing_Linter_Bears.html).
 
 This project proposal should contain a concrete list of linters that are
 wrapped together with concrete improvement proposals to the `@linter`
-decorator if needed. Because implementing a linter bear is a rather trivial
-task it is highly recommended to spice up the project with something
-interesting, like adding project wide linter support to the `@linter`
-decorator or improving the testing API of linter bears or even improving
-the linter bear docs! Improving by adding more options can also be part of this.
+decorator if needed.
+Because implementing a linter bear is a rather trivial task it is highly
+recommended to spice up the project with something interesting, like adding
```

Comment:
Drop "highly recommend" and use "want to" because 3 linter bears is too trivial. Update goals accordingly.